lib/SPIRV/Mangler/ManglingUtils.cpp

```diff
@@ -138,6 +140,7 @@ static const SPIRversion PrimitiveSupportedVersions[PRIMITIVE_NUM] = {
     SPIR12, // PRIMITIVE_IMAGE_2D_ARRAY_DEPTH_T
     SPIR12, // PRIMITIVE_EVENT_T
     SPIR20, // PRIMITIVE_PIPE_T
```

Comment:
Shouldn't these be RO and WO respectively

Reply:
In the comment I mean

Reply:
Fixed